src/System.Runtime.Extensions/src/System/Random.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+/*============================================================
+**
+** Class:  Random
+**
+**
+** Purpose: A random number generator.
+**
+** 
+===========================================================*/
+
+using System;
+using System.Runtime;
+using System.Runtime.CompilerServices;
+using System.Globalization;
+using System.Diagnostics.Contracts;
+
+namespace System
+{
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public class Random
+    {
+        //
+        // Private Constants 
+        //
+        private const int MBIG = Int32.MaxValue;
+        private const int MSEED = 161803398;
+        private const int MZ = 0;
+
+
+        //
+        // Member Variables
+        //
+        private int inext;
+        private int inextp;
+        private int[] SeedArray = new int[56];
```

Comment:
Not listed in the csproj, so I'm not going to touch it, inline with the other stuff.